src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -162,7 +162,6 @@ class __Columns(NamedTuple):
     GESTATIONAL_AGE = "gestational_age"
     ATTENDED_CARE_FACILITY = "attended_care_facility"
     DELIVERY_FACILITY_TYPE = "delivery_facility_type"
-    RECEIVED_ULTRASOUND = "received_ultrasound"
```

Comment:
Not a column in the state table as a result of previous decision tree/state machine work